src/window/replay_confirm.h

```diff
@@ -0,0 +1,6 @@
+#ifndef WINDOW_CONFIRM_REPLAY_H
```

Comment:
I think adding a separate window just for displaying a confirmation dialog is a bit overkill, since we already have the popup_dialog for asking OK/cancel questions.

Reply:
If you ever want to change the popup_dialog for any reason, you will also be changing the replay_mission window. If you ever want to change the replay_mission window then you will then have to make a new file anyways. I'm only following the original design pattern.